README.rst

```diff
@@ -32,6 +32,18 @@ to combine the sources. Additionally, when the original config value is a list,
 operators, which let you extend and modify arrays defined in underlying configurations. See
 https://github.com/kislyuk/tweak#array-merge-operators for a list of these operators.
 
+Service to Service Authorization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For HCA/DCP developers using the HCA-CLI to connect their service, 
+google service credentials can be used to authenticate the service's API requests. The google service credentials must be 
```

Comment:
"The google service account must be whitelisted"

Capitalize Google

"HCA-CLI" -> "HCA CLI"